internal/repos/syncer.go

```diff
@@ -610,6 +611,14 @@ func (s *Syncer) SyncExternalService(
 
 		sourced := res.Repo
 
+		if envvar.SourcegraphDotComMode() && sourced.Private {
+			err := errors.Newf("%s is private, but dotcom does not support private repositories.", sourced.Name)
```

Comment:
```suggestion
			err := errors.Newf("%s is private, but Sourcegraph.com does not support private repositories.", sourced.Name)
```

Or maybe even better, but I don't have a better option :)

Reply:
Here is a screenshot with the `sourcegraph.com` in there

<img width="896" alt="Screenshot 2023-03-14 at 12 23 44" src="https://user-images.githubusercontent.com/9974711/224992822-1ddc5311-5bb8-4b85-99a4-b1f636cf1bdf.png">